pybamm/models/submodels/particle/size_distribution/fickian_single_distribution.py

```diff
@@ -0,0 +1,273 @@
+#
+# Class for a single particle-size distribution representing an
+# electrode, with Fickian diffusion within each particle
```

Comment:
let's leave it for now but I wonder if this class could be merged with the no-size-distribution classes to reduce code duplication